### DISCUSSION TITLE
Tidal I2s formats

### DISCUSSION THREAD

**Discussion by antalpeter** [tidal-music/discussions#20] [Ideas & Feature requests]
I would like to know what are the valid I2S fomats?

**Comment by stefanmoro** [comment]
Hi @antalpeter , and welcome to TIDAL discussions.

It might be that I misunderstand the question here, but to my knowledge, [I2S](https://en.wikipedia.org/wiki/I%C2%B2S) is typically used to communicate PCM between different circuits. TIDAL, being a SW service, does as such not really relate to I2S.

Thank you, and feel free to follow up if I misunderstood the question.

**Reply by antalpeter** [reply]
A Tidal dekódere a külső hangkártyának USB-n keresztül 16 ill 24 bites formátumban közvetíti a soros jelet és ennek sorrendje és Phillips v Japanese formátuma nem ismert
